pcsx2/VMManager.cpp

```diff
@@ -849,19 +849,22 @@ void VMManager::UpdateDiscDetails(bool booting)
 
 		SaveSessionTime(old_serial);
 
+		std::string custom_title = GameList::GetCustomTitleForPath(CDVDsys_GetFile(CDVDsys_GetSourceType()));
```

Comment:
Not really a fan of parsing the INI every time VMManager changes disc.

You could check for a GameList entry matching the path for a faster lookup (probably within `GetCustomTitleForPath()`).